[2022-02-14 18:35:45,439] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.extract_all_data manual__2022-02-14T18:35:43.646639+00:00 [queued]>
[2022-02-14 18:35:45,452] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.extract_all_data manual__2022-02-14T18:35:43.646639+00:00 [queued]>
[2022-02-14 18:35:45,452] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-14 18:35:45,452] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-14 18:35:45,452] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-14 18:35:45,466] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): extract_all_data> on 2022-02-14 18:35:43.646639+00:00
[2022-02-14 18:35:45,472] {standard_task_runner.py:52} INFO - Started process 2648 to run task
[2022-02-14 18:35:45,474] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'extract_all_data', 'manual__2022-02-14T18:35:43.646639+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/dag_01.py', '--cfg-path', '/tmp/tmprnzh8n_o', '--error-file', '/tmp/tmp082rk_ty']
[2022-02-14 18:35:45,475] {standard_task_runner.py:77} INFO - Job 143: Subtask extract_all_data
[2022-02-14 18:35:45,537] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.extract_all_data manual__2022-02-14T18:35:43.646639+00:00 [running]> on host 223810fcea28
[2022-02-14 18:35:45,595] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=extract_all_data
AIRFLOW_CTX_EXECUTION_DATE=2022-02-14T18:35:43.646639+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-14T18:35:43.646639+00:00
[2022-02-14 18:35:52,296] {python.py:175} INFO - Done. Returned value was: None
[2022-02-14 18:35:52,306] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=extract_all_data, execution_date=20220214T183543, start_date=20220214T183545, end_date=20220214T183552
[2022-02-14 18:35:52,355] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-14 18:35:52,375] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
